### ISSUE TITLE
Model instance aggregation "Schema Based Metadata" does not use consistent styling

### ISSUE THREAD

**Issue by hydrocheck** [hydroshare/hydroshare#4248]
The styling can be a bit hard to follow in the "Schema Based Metadata" section - notably, the placement of buttons and whitespace.  In the attached example, the "Collapse" and "Object Properties" for "SWAT Model..." wrap in a somewhat unintuitive way.  On very large width metadata windows, it looks great, however, the default sizing (or similar - up to about 2x the default width) has a lot of wrapping and associated whitespace/styling detials that can be hard to follow.

Possible resolution by opening a separate modal/popup for the Schema Based Metadata, otherwise forcing a wide metadata side panel, or using different styling.

<img width="1920" alt="001" src="https://user-images.githubusercontent.com/43424907/119352226-6a8dc600-bc6f-11eb-975a-54e9ef7db712.png">

**Comment by eclark-cuahsi** [comment]
@horsburgh and @goodalljl I believe this falls under Resource Model -> Model Instance aggregation. Labeling as an enhancement as it pertains to styling.

**Comment by pkdash** [comment]
The text on the green button has been changed to indicate that the same button can be used for hiding or showing schema contents. This change is currently on prod. 

![image](https://user-images.githubusercontent.com/2523434/212403847-dd9c27ef-c316-47db-9ffa-606b9a0309d2.png)

@horsburgh Is this change is sufficient to close this issue?
